treeherder/webapp/api/performance_data.py

```diff
@@ -751,6 +751,29 @@ def _filter_out_retriggers(serialized_data: List[dict]) -> List[dict]:
         return serialized_data
 
 
+class PerformanceAlertTasks(generics.RetrieveAPIView):
+    queryset = PerformanceAlertSummary.objects.prefetch_related(
+        'alerts',
+        'alerts__series_signature',
+        'related_alerts',
+        'related_alerts__series_signature',
+    )
+    serializer_class = PerformanceAlertSummarySerializer
```

Comment:
I believe you should write your own serializer and use it when returning the response, for a detailed example you can take a look at `PerformanceSummary(generics.ListAPIView)` logic.

Reply:
I thought about it when I was developing it, now the API only returns the `task list`, so I didn't make my own serializer for this API. That's why I just reused it :)
I will add it together when the decision is made below review.